tools/UpdateOTS.sh

```diff
@@ -331,6 +331,40 @@ echo -e "UpdateOTS.sh:${LINENO}  1= $1"
 
 if [ "$1"  == "--warn" ]; then #warn should be quiet unless (on stderr) there are uncommitted changes, then output to stderr for capture
 	WARN_ONLY=1
+	echo -e  "\n" >&2 #take stderr for warn result
+
+	#check otsdaq-mu2e-config outside of srcs
+	if [ -d "${OTS_SOURCE}/../otsdaq-mu2e-config" ]; then
```

Comment:
Can this be configurable somehow? These are mu2e-specific, not otsdaq general...